examples/applications/plot_prediction_latency.py

```diff
@@ -93,27 +96,17 @@ def generate_dataset(n_train, n_test, n_features, noise=0.1, verbose=False):
     """Generate a regression dataset with the given parameters."""
     if verbose:
         print("generating dataset...")
+
     X, y, coef = make_regression(n_samples=n_train + n_test,
                                  n_features=n_features, noise=noise, coef=True)
-    X_train = X[:n_train]
-    y_train = y[:n_train]
-    X_test = X[n_train:]
-    y_test = y[n_train:]
-    idx = np.arange(n_train)
-    np.random.seed(13)
-    np.random.shuffle(idx)
-    X_train = X_train[idx]
-    y_train = y_train[idx]
-
-    std = X_train.std(axis=0)
-    mean = X_train.mean(axis=0)
-    X_train = (X_train - mean) / std
-    X_test = (X_test - mean) / std
-
-    std = y_train.std(axis=0)
-    mean = y_train.mean(axis=0)
-    y_train = (y_train - mean) / std
-    y_test = (y_test - mean) / std
+
+    X = StandardScaler().fit_transform(X)
+    y = StandardScaler().fit_transform(y)
```

Comment:
wait a minute. This is buggy.

before the scaling was done using train data even for the test. Now you scale globally which is wrong.

the proper way is to use a pipeline